examples/templates/devcontainer-docker/main.tf

```diff
@@ -89,14 +93,13 @@ EOF
 
 variable "cache_repo" {
   default     = ""
-  description = "Use a container registry as a cache to speed up builds."
-  sensitive   = true
+  description = "(Optional) Use a container registry as a cache to speed up builds."
   type        = string
 }
 
 variable "cache_repo_docker_config_path" {
   default     = ""
-  description = "Path to a docker config.json containing credentials to the provided cache repo, if required."
+  description = "(Optional) Path to a docker config.json containing credentials to the provided cache repo, if required."
   sensitive   = true
```

Comment:
Is this actually sensitive if it's just the path to a file?

Reply:
:shrug: maybe? The nice thing about making the file sensitive is that the content automatically becomes sensitive. (That's my recollection anyhow)

Reply:
Ok, so there’s some magic there wrt content? If so please keep it as is 👍